api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/PaymentRepositoryTest.java

```diff
@@ -45,6 +45,7 @@ void shouldSavePayment() {
     @Test
     void shouldNotSaveAnInvalidPayment() {
         Payment payment = aPaymentWithCardAccountId(null);
+        claimRepository.save(payment.getClaim());
```

Comment:
Now that claim is versioned, hibernate will throw an error about saving a payment with an unsaved claim, before throwing a validation error.